packages/gatsby-cli/src/reporter/reporter.ts

```diff
@@ -83,17 +83,17 @@ class Reporter {
   /**
    * Log arguments and exit process with status 1.
    */
-  panic = (errorMeta: ErrorMeta, error?: Error | Array<Error>): never => {
+  panic(errorMeta: ErrorMeta, error?: Error | Array<Error>): never {
```

Comment:
This change is potentialy breaking. E.g. previously the following would've worked:

```js
const { panic } = reporter
panic(`whatever`)
```

But now it will fail (due to `this` binding in arrows)?